README.md

```diff
@@ -1 +1,87 @@
-# LF_test_task
+<h1> Test task service API </h1>
+
+<h3> REST API version </h3>
+The most recent version of the API is v1.
+
+<h2> Running Docker image <i>(Linux)</i></h2>
+
+In order to run this container you'll need docker [installed](https://docs.docker.com/install/)
+
+In the `project` directory run
+
+```shell
+$ sudo docker build -t my_docker_flask:latest .
```

Comment:
Name should always tell about concrete application - `my_docker_flask` is very vague.